doc/api_assets/js-flavor-cjs.svg

```diff
@@ -2,4 +2,4 @@
  * Font Awesome Free 5.15.2 by @fontawesome - https://fontawesome.com
  * License - https://fontawesome.com/license/free - CC BY 4.0
 -->
-<svg xmlns="http://www.w3.org/2000/svg" width="2719" height="384"><path d="M1191.326 384h192c106 0 192-86 192-192s-86-192-192-192h-192c-106 0-192 86-192 192s86 192 192 192zm0-320c70.8 0 128 57.3 128 128 0 70.8-57.3 128-128 128-70.8 0-128-57.3-128-128 0-70.8 57.3-128 128-128z"/><text stroke-width="42" font-family="sans-serif" font-weight="lighter" font-size="490" y="370">CJS</text><text stroke-width="42" font-weight="lighter" font-family="sans-serif" font-size="490" y="370" x="1682">ESM</text></svg>
+<svg xmlns="http://www.w3.org/2000/svg" height="384" width="2719"><path d="M1191 384h192a192 192 0 0 0 0-384h-192a192 192 0 0 0 0 384zm0-320a128 128 0 1 1 0 256 128 128 0 0 1 0-256z"/><path d="M20 196q0-25 4-49t16-50q11-26 29-45 17-20 47-32 30-13 67-13 116 0 137 117h-46q-8-39-31-58t-66-19q-52 0-82 40-30 39-30 108 0 67 32 107 31 39 84 39 44 0 68-24 23-25 32-77h47q-16 141-147 141-37 0-66-12-29-13-47-32-17-19-29-44-11-26-15-50-4-23-4-47Zm445 147q31 0 43-21 11-20 11-58V13h46v268q0 46-27 73t-74 27q-46 0-73-25-26-26-26-69v-32h46v23q0 32 14 48 14 17 40 17zm308-2q28 0 47-6 19-7 28-17t13-20q4-10 4-22 0-40-66-57l-89-24q-70-18-70-83 0-49 34-77t93-28q62 0 96 29t35 82h-43q-1-35-24-54t-65-19q-37 0-59 17-21 16-21 43 0 21 13 33 13 11 45 20l90 24q36 10 56 34 20 23 20 56 0 14-4 28-3 14-13 29t-25 26q-15 12-41 19-25 7-57 7-20 0-38-3t-38-11q-19-9-32-22-14-14-23-37-9-22-10-52h43v3q0 15 5 28t15 26q11 13 31 21 21 7 50 7z" aria-label="CJS"/><path d="M1767 207v123h210v40h-256V13h248v40h-202v114h194v40zm393 134q27 0 46-6 19-7 28-17t13-20q4-10 4-22 0-40-66-57l-89-24q-70-18-70-83 0-49 34-77t93-28q62 0 96 29 35 29 35 82h-43q0-35-24-54-23-19-65-19-36 0-58 17-22 16-22 43 0 21 13 33 13 11 46 20l89 24q37 10 56 34 20 23 20 56 0 14-4 28-3 14-13 29t-25 26q-15 12-40 19t-57 7q-20 0-39-3-18-3-37-11-19-9-33-22-14-14-23-37-9-22-9-52h43v3q0 15 4 28 5 13 15 26 11 13 32 21 21 7 50 7zm391 29h-48L2402 71v299h-43V13h63l106 311 104-311h63v357h-43V71Z" aria-label="ESM"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" height="384" width="2719"><path d="M1191 384h192a192 192 0 0 0 0-384h-192a192 192 0 0 0 0 384zm0-320a128 128 0 1 1 0 256 128 128 0 0 1 0-256z"/><path d="M20 196q0-25 4-49t16-50q11-26 29-45 17-20 47-32 30-13 67-13 116 0 137 117h-46q-8-39-31-58t-66-19q-52 0-82 40-30 39-30 108 0 67 32 107 31 39 84 39 44 0 68-24 23-25 32-77h47q-16 141-147 141-37 0-66-12-29-13-47-32-17-19-29-44-11-26-15-50-4-23-4-47Zm445 147q31 0 43-21 11-20 11-58V13h46v268q0 46-27 73t-74 27q-46 0-73-25-26-26-26-69v-32h46v23q0 32 14 48 14 17 40 17zm308-2q28 0 47-6 19-7 28-17t13-20q4-10 4-22 0-40-66-57l-89-24q-70-18-70-83 0-49 34-77t93-28q62 0 96 29t35 82h-43q-1-35-24-54t-65-19q-37 0-59 17-21 16-21 43 0 21 13 33 13 11 45 20l90 24q36 10 56 34 20 23 20 56 0 14-4 28-3 14-13 29t-25 26q-15 12-41 19-25 7-57 7-20 0-38-3t-38-11q-19-9-32-22-14-14-23-37-9-22-10-52h43v3q0 15 5 28t15 26q11 13 31 21 21 7 50 7z" aria-label="CJS"/><path d="M1767 207v123h210v40h-256V13h248v40h-202v114h194v40zm393 134q27 0 46-6 19-7 28-17t13-20q4-10 4-22 0-40-66-57l-89-24q-70-18-70-83 0-49 34-77t93-28q62 0 96 29 35 29 35 82h-43q0-35-24-54-23-19-65-19-36 0-58 17-22 16-22 43 0 21 13 33 13 11 46 20l89 24q37 10 56 34 20 23 20 56 0 14-4 28-3 14-13 29t-25 26q-15 12-40 19t-57 7q-20 0-39-3-18-3-37-11-19-9-33-22-14-14-23-37-9-22-9-52h43v3q0 15 4 28 5 13 15 26 11 13 32 21 21 7 50 7zm391 29h-48L2402 71v299h-43V13h63l106 311 104-311h63v357h-43V71Z" aria-label="CJS"/></svg>
```

Reply:
That doesn't look right to me. I double checked by opening the SVG in Inkscape and the `aria-label` values match correctly for me. Additionally, this change would make both text labels have the same `aria-label` value.

Reply:
Double checked just now and see that there are two labels in a single image (one per path).

I stand corrected, but this doesn't feel like how to make an accessible toggler to me.